cmd/fleet/cron.go

```diff
@@ -719,8 +720,14 @@ func newCleanupsAndAggregationSchedule(
 		schedule.WithJob(
 			"distributed_query_campaigns",
 			func(ctx context.Context) error {
-				_, err := ds.CleanupDistributedQueryCampaigns(ctx, time.Now())
-				return err
+				_, recentInactiveIDs, err := ds.CleanupDistributedQueryCampaigns(ctx, time.Now().UTC())
```

Comment:
Pretty sure this missing `Now().UTC()` was a silent bug not behaving as we expected - the timestamps in the DB are UTC and in the tests we use `mockclock.New()` which creates a UTC-based clock.